osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/EffectPointVisualisation.cs

```diff
@@ -68,5 +68,8 @@ private void load()
                 }
             }, true);
         }
+
+        // kiai sections display duration, so are required to be visualised.
+        public bool IsRedundant(ControlPoint other) => (other as EffectControlPoint)?.KiaiMode == effect.KiaiMode;
```

Comment:
no type check? so any other point type can hide this? may be especially relevant for scroll speed changes, which are handled by effect points on scrolling rulesets now

Reply:
I don't quite follow. The existing `as` check will cause a null/false if the type doesn't match.

Reply:
oh. i was defeated by nullable tri-state boolean logic